home-assignments/session2/cli.py

```diff
@@ -0,0 +1,108 @@
+""""
+Ze'ev 10/11/2018
+Exercises for the OOPS School 3, Coding in Python, Session 2, Assignment 1
+
+Command line arguments follow the following format -
+    `python cli.py [] CITY`
+CITY is a mandatory argument
+Options:
+    [--period <num-of-days> -f]
+num-of-days may be in format TODAY+n
+num-of-days default value is TODAY == 1
+Use -f to override default to Fahrenheit measurements
+"""
+
+import click
+import requests
+from weather import Weather, Unit
+
+unit_dict = {
```

Comment:
This is nice but could be made much simpler:

```python
UNIT_DICTIONARY = {
    "Celsius": Unit.CELSIUS,
    "Fahrenheit": Unit.FAHRENHEIT
}
```

or even better use `getattr(Unit, the_unit_string)`

Reply:
I think using classes in this use case is the overkill. I didn't meet before the const expression as you propose, but will try to accomodate.

Reply:
I understood what you meant, but I implemented slightly different logic. -f (Fahrenheint) is defined as boolean at cl.